From 8f3507c2ebd02cf9f08995b35a8d78da0584450c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 10 Feb 2025 11:03:30 +0800
Subject: [PATCH 1/2] AOSCOS: chore: loosen dependency requirements

So that we can more easily update dependencies - and we will test it too.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 sass-listen.gemspec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sass-listen.gemspec b/sass-listen.gemspec
index 7844632..82cce42 100644
--- a/sass-listen.gemspec
+++ b/sass-listen.gemspec
@@ -22,8 +22,8 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 1.9.3'
 
-  s.add_dependency 'rb-fsevent', '~> 0.9', '>= 0.9.4'
-  s.add_dependency 'rb-inotify', '~> 0.9', '>= 0.9.7'
+  s.add_dependency 'rb-fsevent', '>= 0.9', '>= 0.9.4'
+  s.add_dependency 'rb-inotify', '>= 0.9', '>= 0.9.7'
 
   s.add_development_dependency 'bundler', '>= 1.3.5'
 end
-- 
2.48.1

